src/widgets/topicFrequencyGraph.js

```diff
@@ -71,4 +91,7 @@ export default async function topicFrequencyGraph(scopes) {
 
     return acc;
   }, topicsResponse);
+
+  // console.timeEnd('overallTime2');
```

Comment:
remove console log?

Reply:
I cleaned this up to remove the timings. I also added back the approved status in the where.